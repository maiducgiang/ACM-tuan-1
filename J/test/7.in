19
752,156(558)
110,724(719)
485,821(551)
866,466(934)
860,855(234)
358,264(116)
365,353(190)
535,516(806)
541,491(775)
927,203(617)
909,332(675)
614,648(270)
193,378(628)
847,761(589)
374,832(927)
623,163(460)
181,423(765)
688,317(737)
976,140(118)
