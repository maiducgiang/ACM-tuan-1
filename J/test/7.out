41744689 55500
22122799 199800
48533573 99900
216400117 249750
95554931 111000
89476463 249750
364987837 999000
53498129 99900
135237571 249750
463138207 499500
33645309 37000
11370993 18500
772741 3996
211728457 249750
8321291 22200
622540297 999000
10069019 55500
34381471 49950
487581989 499500
